modules/i3s/test/parse-slpk.spec.js

```diff
@@ -0,0 +1,30 @@
+import test from 'tape-promise/tape';
+import {load} from '@loaders.gl/core';
+import {SlpkLoader} from '../src';
+import LocalFileHeader from '../src/lib/parsers/parse-slpk/local-file-header';
+import {DATA_ARRAY} from './data/test.zip.js';
+import CDFileHeader from '../src/lib/parsers/parse-slpk/cd-file-header';
+
+test('SlpkLoader#slpk load', async (t) => {
+  const slpkUrl = '@loaders.gl/i3s/test/data/DA12_subset.slpk';
+  const uncompressedFile = await load(slpkUrl, SlpkLoader, {path: 'nodepages/0.json'});
+  t.deepEqual(uncompressedFile.byteLength, 16153);
+  t.end();
+});
+
+test('SlpkLoader#local file header parce', async (t) => {
```

Comment:
make individual files to test these classes `test/parsers/parse-slpk/cd-file-header.spec.js`, `test/parsers/parse-slpk/local-file-header.spec.js`